admin/scripts/modules/reporty/reporty.php

```diff
@@ -11,28 +11,29 @@
-  ['zaplnenost-programu-ucastniku', 'Zaplněnost programu účastníků (2015)', ['csv', 'html']],
-  ['maily-prihlaseni',  'Maily – přihlášení na GC (vč. unsubscribed)', ['csv', 'html']],
-  ['maily-neprihlaseni','Maily – nepřihlášení na GC',             ['csv', 'html']],
-  ['maily-vypraveci',   'Maily – vypravěči (vč. unsubscribed)',   ['csv', 'html']],
-  ['maily-vsichni',     'Maily – všichni',                        ['csv', 'html']],
-  ['celkovy-report',    '<br>Celkový report '.ROK.'<br><br>',     ['csv', 'html']],
+  ['aktivity',                              'Historie přihlášení na aktivity',              ['csv', 'html']],
+  //['neplatici',                           'Neplatiči letos'], // duplikuje bfgr a finance, nemá smysl udržovat
+  //['spolupracovnici',                     'Spolupracovníci (orgové, info, zázemí, vypravěči+aktivity)'], // neudržované, asi lze zjistit z bfgr
+  ['pocty-her',                             'Účastníci a počty jejich aktivit'],
+  ['pocty-her-graf',                        'Graf rozložení rozmanitosti her',              ['html']],
+  ['rozesilani-ankety',                     'Rozesílání ankety s tokenem',                  ['html']],
+  ['parovani-ankety',                       'Párování ankety a údajů uživatelů',            ['html']],
+  ['grafy-ankety',                          'Grafy k anketě',                               ['html']],
+  ['update-zustatku',                       'UPDATE příkaz zůstatků pro letošní GC',        ['html']],
+  ['ubytovani',                             'Ubytování',                                    ['csv', 'html']],
+  //['celkova-ucast',                       'Celková účast'], // už dlouho nefunkční, asi přehled účasti po letech
+  ['neprihlaseni-vypraveci',                'Nepřihlášení a neubytovaní vypravěči',         ['html']],
+  ['duplicity',                             'Duplicitní uživatelé',                         ['html']],
+  ['stravenky',                             'Stravenky uživatelů',                          ['html']],
+  ['stravenky?ciste',                       'Stravenky (bianco)',                           ['html']],
+  ['programove-reporty',                    'Programový report (2015)',                     ['csv', 'html']],
+  ['zaplnenost-programu-ucastniku',         'Zaplněnost programu účastníků (2015)',         ['csv', 'html']],
+  ['maily-prihlaseni',                      'Maily – přihlášení na GC (vč. unsubscribed)',  ['csv', 'html']],
+  ['maily-neprihlaseni',                    'Maily – nepřihlášení na GC',                   ['csv', 'html']],
+  ['maily-vypraveci',                       'Maily – vypravěči (vč. unsubscribed)',         ['csv', 'html']],
+  ['maily-dle-data-ucasti?start=0',         'Maily - nedávní účastníci (prvních 2000)'],
+  ['maily-dle-data-ucasti?start=2000',      'Maily - dávní účastníci (dalších 2000)'],
+  ['celkovy-report',                        '<br>Celkový report '.ROK.'<br><br>',           ['csv', 'html']],
```

Comment:
Sry za nepřesné info - myslel jsem zarovnat jen ty nové změny. Zapomněl jsem napsat do style guide, že řádky by neměly překročit 80, je to jen v PSR. Teď už to neměň, toto zarovnání je stále OK, jen do budoucna pro info.

Reply:
A jak obecně zarovnávat pole s dlouhými hodnotami, pokud překročí 80 znaků?

Reply:
Pokud to překročí 80 znaků po zarovnání, bez zarovnání je však ok, a většina ostatních řádků je zarovnaná, pak řádek nezarovnávat (tj. hodnoty oddělené standardně pomocí `, `) a nechat jako výjimku.

Pokud už začne být výsledek špatně čitelný (řádek je moc dlouhý i bez zarovnání, moc řádků překračuje limit), zapíše se pole pod sebe. V takovém případě je lepší mít čárku i za poslední položkou, protože přidání položky pak nevyžaduje přidávání čárky na řádek, který de facto měnit nechceš.